okio/src/commonMain/kotlin/okio/BufferedSource.kt

```diff
@@ -268,6 +268,37 @@ expect sealed interface BufferedSource : Source {
    */
   fun select(options: Options): Int
 
+  /**
+   * Finds the first item in [options] whose encoding is a prefix of this buffer, consumes it from
+   * this buffer, and returns it. If no item in [options] is a prefix of this source, this function
+   * returns null and no bytes are consumed.
+   *
+   * This can be used as an alternative to [readByteString] or even [readUtf8] if the set of
+   * expected values is known in advance.
+   *
+   * ```
```

Comment:
For nice highlights
```suggestion
   * ```java
```

Reply:
I’m gonna save this in follow-up for all the code examples.

Reply:
https://github.com/square/okio/pull/1445